benchmark/src/main/scala/fs2/benchmark/ConcurrentBenchmark.scala

```diff
@@ -2,20 +2,22 @@ package fs2
 package benchmark
 
 import cats.effect.{ContextShift, IO}
-import org.openjdk.jmh.annotations.{Benchmark, Scope, State}
+import org.openjdk.jmh.annotations.{Benchmark, Param, Scope, State}
 
 @State(Scope.Thread)
 class ConcurrentBenchmark {
   implicit val cs: ContextShift[IO] =
     IO.contextShift(scala.concurrent.ExecutionContext.Implicits.global)
 
-  @GenerateN(1, 2, 4, 7, 16, 32, 64, 128, 256)
+  @Param(Array("1", "16", "256"))
```

Comment:
Curious as to why you lowered the number of possibilities here

Reply:
Running the JMH suite takes hours and I don't typically look at many of the intermediate results -- just compare the individual cases enough to get a feel for asymptotic growth.